src/Core/src/Core/ISwipeItemView.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Microsoft.Maui
+{
+	public interface ISwipeItemView : IContentView, ISwipeItem
```

Comment:
I think we should have a team meeting reviewing this point. SwipeItemView is  SwipeItem that could contain a View (anything). Is implemented in Android and iOS in Xamarin.Forms but NOT in Windows. In Windows we use https://docs.microsoft.com/es-es/windows/winui/api/microsoft.ui.xaml.controls.swipecontrol?view=winui-3.0 and not allow this scenario. So we have two approaches:
- Change the Windows implementation creating our own MauiSwipeView and allow to use SwipeItemView everywhere.
- Remove the SwipeItemView support and align the functionality to common denominator.

Reply:
Can we do option one but create a tracking issue for it?
I don't think we want to diminish the behavior here for users upgrading from XF to .NET MAUI

Reply:
I am going to create the issue to track it.